system/mp/test/mptest.cpp

```diff
@@ -742,78 +887,124 @@ int main(int argc, char* argv[])
         else
         {
             while (i+1 < argc && i-1 < 1000) {
-                PrintLog("MPTEST: adding node %d, port = <%s>", i-1, argv[i+1]);
+                PrintLog("MPTEST: adding node %u, port = <%s>", i-1, argv[i+1]);
                 nodes[i-1] = createINode(argv[i+1], my_port);
                 i++;
             }
         }
 
-        PrintLog("MPTEST: num_nodes = %d", i-1);
+        tot_ranks = i-1;
+
+        IGroup *group = createIGroup(tot_ranks, nodes);
+
+        // stop if not meant for this host ...
+
+        IpAddress myIp;
+        GetHostIp(myIp);
+        SocketEndpoint myEp(my_port, myIp);
+#if 0
+        StringBuffer myurlStr;
+        myEp.getUrlStr(myurlStr);
+        PrintLog("MPTEST: myEp %s", myurlStr.str());
+#endif
```

Comment:
can we delete the #if 0 blocks, looks like testing code not for release.
or could combine myEp tracing into line 926